runtime/stdlib/builtin.go

```diff
@@ -34,3 +34,32 @@ var BuiltinValues = []StandardLibraryValue{
 	//   (BLSVerifyPoPHandler, BLSAggregateSignaturesHandler, BLSAggregatePublicKeysHandler)
 	BLSContract,
 }
+
+type StandardLibraryHandler interface {
+	Logger
+	UnsafeRandomGenerator
+	BlockAtHeightProvider
+	CurrentBlockProvider
+	PublicAccountHandler
+	AccountCreator
+}
+
+func DefaultStandardLibraryValues(handler StandardLibraryHandler) []StandardLibraryValue {
+	return append(
+		BuiltinValues[:],
+		NewLogFunction(handler),
+		NewUnsafeRandomFunction(handler),
+		NewGetBlockFunction(handler),
+		NewGetCurrentBlockFunction(handler),
+		NewGetAccountFunction(handler),
+		NewAuthAccountConstructor(handler),
+	)
+}
+
+// TODO: use in language server
```

Comment:
Maybe better to make this TODO a issue in the tools repo, since we might miss out to remove it from here, once this is implemented in the other repo

Reply:
Created onflow/cadence-tools#21